src/Symfony/Component/HttpKernel/DataCollector/ConfigDataCollector.php

```diff
@@ -242,4 +258,59 @@ public function getName()
     {
         return 'config';
     }
+
+    /**
+     * Tries to retrieve information about the current Symfony version.
+     *
+     * @return string One of: unknown, dev, stable, eom, eol
+     */
+    private function requestSymfonyState()
+    {
+        // get version information from cache or the roadmap
+        $versionCachePath = $this->kernel->getCacheDir().'/version_info.json';
+        if (file_exists($versionCachePath)) {
+            $versionInfo = json_decode(file_get_contents($versionCachePath), true);
+        } else {
+            $versionResponse = @file_get_contents('http://symfony.com/roadmap.json?version='.$this->data['symfony_version']);
+
+            if (false !== $versionResponse) {
+                $versionInfo = json_decode($versionResponse, true);
+            }
+        }
+
+        // get the version state
+        $versionState = 'unknown';
+        if (isset($versionInfo)) {
```

Comment:
Instead of using isset here, I would prefer that we always define the `$versionInfo` variable and test for `null` here.